src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  handleClick: (arg: string)=>void,
+};
+
+export const Tabs:React.FC<Props> = (
+  {
+    tabs,
+    selectedTabId,
+    handleClick,
+  },
+) => {
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${selectedTabId}`}
+      </h1>
+
+      <div>
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                className={classNames(
+                  { 'is-active': selectedTabId === tab.id },
+                )}
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  id={tab.id}
+                  onClick={(e) => {
+                    e.preventDefault();
```

Comment:
prevent default is redundant here, this event don't have specific actions on click (as submit event, for example)